### ISSUE TITLE
Help needed

### ISSUE THREAD

**Issue by stepanov1975** [custom-components/hacs#21]
Sorry but I still don't understand how to migrate from cusrtom_updater
I have a couple of custom elements. For example https://github.com/custom-cards/button-card
Is it possible to manage it using HACS? If yes how?
I tried to add repo as custom plugin, but got error
```
Could not add repo 'https://github.com/custom-cards/button-card' at this time, if the repo meet all requirements try again later.
```

**Comment by ludeeus** [comment]
Everything that follows the requirements can be added.
Elements in https://github.com/custom-cards and https://github.com/custom-components (that follow the requirements) are added by default.

'custom-cards/button-card' _should_ work, but it does not.
Will fix that to 0.3.0 so look out for that one.

**Comment by ludeeus** [comment]
This has been resolved in https://github.com/custom-components/hacs/releases/tag/0.4.0
